backend/migrate_photo_urls_full.sh

```
@@ -0,0 +1,211 @@
+RED='\033[0;31m'
+GREEN='\033[0;32m'
+YELLOW='\033[1;33m'
+BLUE='\033[0;34m'
+NC='\033[0m'
+
+CONTAINER_NAME="suitesme_postgres_1"
+DB_NAME="postgres"
+DB_USER="postgres"
+
+# Old patterns to fix
+OLD_SERVER_PATTERN_1="http://51.250.84.195:9000"
```

Shellcheck (0.11.0)

[warning] OLD_SERVER_PATTERN_1 appears unused. Verify use (or export if used externally).

(SC2034)